### PR TITLE
[Calculate] adding works

### PR DESCRIPTION
**Pull request by Crazyayane07**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: calculate
🏅 Granted 6 XP for @Crazyayane07

File: homework/calculate/calculate.hpp [hunk 1]
```diff
@@ -2,6 +2,33 @@
...
+	{
+		result = first - second;
+		return std::to_string(result);
+	}
+	else if (command == "multiply")
+	{
+		result = first * second;
+		return std::to_string(result);
+	}
+	else if (command == "divide")
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:22]: If you return variable here, you don't need to use `else if`

File: homework/calculate/calculate.hpp [hunk 2]
```diff
@@ -2,6 +2,33 @@
 #include <string>
 
 std::string calculate(const std::string& command, int first, int second) {
-    // TODO: Implement your solution here and return proper value
-    return "";
+	int result;
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:5]: Unnecessary.

File: homework/calculate/calculate.hpp [hunk 3]
```diff
@@ -2,6 +2,33 @@
...
+
+	if (command == "add") 
+	{
+		result = first + second;
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:9]: Why?
Just `return std::to_string(result);`

File: homework/calculate/calculate.hpp [hunk 4]
```diff
@@ -2,6 +2,33 @@
...
+	}
+	else if (command == "divide")
+	{
+        if(second == 0)
+        {
+            return "Division by 0";
+        }
+
+		result = first / second;
+		return std::to_string(result);
```
Comment (nauka-programowania-MA) [inline homework/calculate/calculate.hpp:30]: Wrong indentation

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**